ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -280,7 +289,9 @@ public void shouldReturnCorrectSchemaForPullQueryWithOnlyValueColumnInSelect() {
     final List<String> messages = assertThatEventually(call, hasSize(HEADER + 1));
     assertValidJsonMessages(messages);
     assertThat(messages.get(0),
-        is("[{\"name\":\"COUNT\",\"schema\":{\"type\":\"BIGINT\",\"fields\":null,\"memberSchema\":null}}]"));
+        is("["
+            + "{\"name\":\"COUNT\",\"schema\":{\"type\":\"BIGINT\",\"fields\":null,\"memberSchema\":null}}"
+            + "]"));
     assertThat(messages.get(1),
         is("{\"row\":{\"columns\":[1]}}"));
   }
```

Comment:
Should you add tests here also for push queries? `shouldReturnCorrectSchemaForPushQuery...`